packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -396,6 +396,7 @@ describe('plugin-meetings', () => {
             mediaConnections: 'mediaConnections',
           },
         });
+        parseLocusJoinSpy.restore();
```

Comment:
Why is this restore required if we are creating the spy in every test?

Reply:
Same as above.